docs/reference/superjson.mdx

```diff
@@ -0,0 +1,294 @@
+---
+slug: /reference/superjson
+---
+
+# Super.json Reference
+
+Super.json is the main Superface configuration file. It contains the configuration about profiles, maps and providers used.
```

Comment:
Preferably avoid capitalization (same for title), the file is called `super.json` and most apps will likely run on case-sensitive FS. :wink: 

```suggestion
`super.json` is the main Superface configuration file. It specifies which profiles, maps, and providers are used.
```

Maybe also mention where the file needs to be located (in `superface` directory).